doc/source/ray-air/examples/gptj_batch_prediction.ipynb

```diff
@@ -63,6 +63,7 @@
     "        \"pip\": [\n",
     "            \"accelerate>=0.16.0\",\n",
     "            \"transformers>=4.26.0\",\n",
+    "            \"numpy<1.24\",  # remove when mlflow updates beyond 2.2\n",
```

Comment:
Don't think this is actually needed if it's just for compatibility with `ray-ml` Docker images (since they now have `mlflow` 1.30.0 which is also compatible).

Reply:
Unfortunately this example breaks without this on Anyscale. Do you have a link to when this change was implemented?

1. If it was done in nightly, then we should merge this change so that we can still run gpt-j examples before the next release
2. If it was done before 2.3, then MLflow is a red herring